### PR TITLE
Vox and Plasmaman helmets no longer protect from flashes

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
Removes EYEPROT from civilian Vox and Plasmaman helmets (e.g. all but engineering and security). This will make them vulnerable to flashing despite being space helmets. Please note that this also removes their welder protection too, so please remember to wear welding goggles if you need to weld something and aren't one of those jobs at roundstart.

Addresses #6093

### CODE REVIEW

**Comment by 9600bauds** [comment]
:+1:

**Comment by Exxion** [comment]
:+1:, but changelog please

**Comment by Ephx1** [comment]
:+1: all the waaaaaaaaaay

**Comment by ihadtoregisterforthis** [comment]
Also make it close the issues about this.

**Comment by Myrios** [comment]
On a side note, Vox Engineer space helmets do not protect from welder glare.

**Comment by CptWad** [comment]
@ihadtoregisterforthis add the issues in the title, it will automatically close them when it merges

**Comment by Exxion** [comment]
@CptWad The title actually doesn't work for that
Has to be in the body of the OP or in a commit message

**Comment by Imagebard** [comment]
:+1:

**Comment by icantthinkofanameritenow** [comment]
@ihadtoregisterforthis 
Which issues would those be?

**Comment by CptWad** [comment]
:+1:

**Comment by ghost** [comment]
It partially addresses that issue, but yeah, :+1:

**Comment by Arthurdentist** [comment]
:+1:

**Comment by FudgePucker** [comment]
:+1: remove xenos
Human pride station wide.
Also shitters who pick plasmashits or birdshits then griff sec by abusing the immunity to wall flashes are the fucking worse

**Comment by Imagebard** [comment]
>shitters who pick plasmashits or birdshits then griff sec by abusing the immunity to wall flashes are the fucking worse

They're the best.
If they start breaking shit and have flash protection I usually grab a weapon form the armory and threaten them with lethals.

**Comment by icantthinkofanameritenow** [comment]
So, why hasn't this been merged yet?

**Comment by ghost** [comment]
It was made one day ago, nerd.

**Comment by Exxion** [comment]
Finally

**Comment by 9600bauds** [comment]
a new age

**Comment by Intigracy** [comment]
vox engineer and vox sec helmets don't, but plasmaman engineer and plasmaman sec helmets do..............

**Comment by CptWad** [comment]
I tested that too and I think thats intended as vox are supposed to be shit or something
I guess either nerf plasman or buff vox if its a problem

**Comment by Intigracy** [comment]
In the case of engineers, they can't even use the suit helmet to weld without fucking up their eyes.

**Comment by CptWad** [comment]
I think I made a bugreport eons ago of that lemme look it up

**Comment by Intigracy** [comment]
It had eyeprot before this PR.

He even re-added it for plasmamen sec/engis, just not for vox.

**Comment by icantthinkofanameritenow** [comment]
Did I? I'm pretty sure Vox Sec and engi gear still had it when I coded it.

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -96,6 +97,7 @@
 	base_state = "plasmamanEngineer_helmet"
 	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 35, bio = 100, rad = 80)
 	pressure_resistance = 200 * ONE_ATMOSPHERE
+	eyeprot = 1
```
Comment (Intigracy) [inline code/modules/clothing/spacesuits/plasmamen.dm:100]: You gave plasman engineer/security eyeprot = 1 specifically

https://github.com/d3athrow/vgstation13/pull/8630/files#diff-2c1ddf967eb855c5c3735353295c3eb0R198

but not vox

**Pull request merged**